.dev/CRAN_Release.cmd

```diff
@@ -95,16 +95,18 @@ grep "PROTECT_PTR" ./src/*.c
 # No use of long long, instead use int64_t. TODO
 # grep "long long" ./src/*.c
 
-// No use of llu, lld, zd or zu
-grep -nE "(llu|lld|zd|zu)" src/*.[hc]
+// No use of llu, lld, td or tu
+grep -nE "%(ll|t)[ud]\b" src/*.[hc]
 // Comment moved here from fread.c on 19 Nov 2019
 // [Moved from fread.c on 19 Nov 2019] On Windows variables of type `size_t` cannot be printed
-// with "%zu" in the `snprintf()` function. For those variables we used to cast them into
-// `unsigned long long int` before printing, and defined (llu) to make the cast shorter.
-// We're now observing warnings from gcc-8 with -Wformat-extra-args, #4062. So
-// now we're more strict and cast to [u]int64_t and use PRIu64/PRId64 from <inttypes.h>
-// In many cases the format specifier is passed to our own macro (e.g. DTPRINT) or to Rprintf(),
-// error() etc, and even if they don't call sprintf() now, they could in future.
+//   with "%zu" in the `snprintf()` function. For those variables we used to cast them into
```

Comment:
Hmm, it seems we've already let plenty of `%zu` slip through, e.g.

https://github.com/Rdatatable/data.table/blob/7620b469f7c9f24eb95aebc362d11fe1b90f9f7e/src/reorder.c#L22

So I've just dropped that check. I also see som `%lld` have slipped through, e.g.

https://github.com/Rdatatable/data.table/blob/7620b469f7c9f24eb95aebc362d11fe1b90f9f7e/src/fifelse.c#L260

Is this check just obsolete?